DigitalInnovationHub/schema.json

```diff
@@ -27,16 +27,16 @@
           "description": "Property. [URL](https://schema.org/URL). Website of the Digital Innovation Hub."
```

Comment:
why this change? it is also valid, and the previous model was more general. Need justification.